### PR TITLE
Add support for SHA checking of downloaded file.

### PR DESCRIPTION
**Pull request by troymohl**
Added SHA1 checking of downloaded .zip file. 
Added the ability to specify the repo on Nexus.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
+ENV STARTUP_APPS=admin-app;platform-app;catalog-app;spatial-app;security-services-app;security-app
+ENV UNINSTALL_FEATURES=admin-modules-installer
 
 RUN apk add --no-cache curl unzip
 
 RUN mkdir -p $APP_BASE/tmp \
     && mkdir -p $APP_HOME \
     && cd $APP_BASE/tmp \
-    && curl -o alliance.zip "http://artifacts.codice.org/service/local/artifact/maven/content?g=org.codice.alliance.distribution&a=alliance&v=$APP_VERSION&r=releases&e=zip" \
+    && curl -o alliance.sha1 "http://artifacts.codice.org/service/local/artifact/maven/content?g=org.codice.alliance.distribution&a=alliance&v=$APP_VERSION&r=$APP_REPO&e=zip.sha1" \
```
Comment (oconnormi) [inline Dockerfile:18]: Really like the checksum addition here 👍

File: Dockerfile [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
+FROM oconnormi/ddf-base
 MAINTAINER oconnormi
 
 ENV APP_VERSION=0.2
 ENV APP_BASE=/opt
 ENV APP_HOME=$APP_BASE/alliance
 ENV APP_LOG=$APP_HOME/data/log/ddf.log
+#releases, snapshots
+ENV APP_REPO=releases
+ENV STARTUP_APPS=admin-app;platform-app;catalog-app;spatial-app;security-services-app;security-app
```
Comment (oconnormi) [inline Dockerfile:10]: Just wanted to double check with you that these apps work correctly

File: Dockerfile [hunk 3]
```diff
@@ -1,17 +1,25 @@
...
+ENV UNINSTALL_FEATURES=admin-modules-installer
```
Comment (oconnormi) [inline Dockerfile:11]: Do the other admin modules get installed, i.e. `configuration`, `sources`...?
I can't remember if the apps do it or if you need to have the `admin-post-install-modules` install instead

Comment (troymohl) [reply Dockerfile:11]: Default install appears to check out

**Review by oconnormi — APPROVED** [review]

**Pull request merged**